### DISCUSSION TITLE
How to troubleshoot windows flannel cni

### DISCUSSION THREAD

**Discussion by jvnvenu** [rancher/rke2#6865] [General]
we have a application which is running in a windows node. 

The application is trying to connect a external Service. But there is no traffic received on the destination machine. But ping and telnet is working.

So we want to know how to troubleshoot the flannel network in windows node

**Comment by manuelbuil** [comment]
By external service you mean a service which is not served by the k8s cluster?

**Comment by jvnvenu** [comment]
yes it is running in a standalone machine.

**Comment by manuelbuil** [comment]
You can start by running things like `get-hnsnetwork` and verify the network is there and `get-hnsendpoint` to check if the pod endpoints are present. Another test is querying `get-hnsnamespace` and verify the namespace specified in the endpoint exists.

**Comment by jvnvenu** [comment]
i could see all 3 created

PS C:\Users\Administrator> get-hnsnetwork | select name

Name
----
flannel.4096

PS C:\Users\Administrator> get-hnsendpoint | select name

Name
----
0c00b99e93312d5699a74b4734afc36cf8bd3963eb755fa00622a4758f2a72c0_flannel.4096
048fdbfdfd446599ac3b01869c44fcaa7d6be183690f29baf7ab100646e9bd8d_flannel.4096
1a9e64510535e8f0ee350154c2fee38da0dd98ff0dc29c51d532cdc2e83b857b_flannel.4096

PS C:\Users\Administrator> get-hnsnamespace | select id

ID
--
EA3E2DA6-33C7-4B53-8DA0-A7FA907E5437
EF13AECB-4DF6-4C06-94BB-9556EB6CEDFC
222082A3-B1FA-41B2-8CAB-9538BC057CDD
